deploy/roles/postgresql-server/templates/postgresql.conf.96.j2

```diff
@@ -0,0 +1,637 @@
+
+#oracle_home ='{{ pp_oracle_home }}'
+# path to the Oracle home directory;
+# only used by OCI Dblink; defaults
+# to ORACLE_HOME environment variable.
+
+#------------------------------------------------------------------------------
+# LOCK MANAGEMENT
+#------------------------------------------------------------------------------
+
+deadlock_timeout = {{ pp_deadlock_timeout }}
+max_locks_per_transaction = {{ pp_max_locks_per_transaction }} # min 10 (change requires restart)
+
+# Note:  Each lock table slot uses ~270 bytes of shared memory, and there are
+# max_locks_per_transaction * (max_connections + max_prepared_transactions)
+# lock table slots.
+max_pred_locks_per_transaction = {{ pp_max_pred_locks_per_transaction }}	# min 10 (change requires restart)
+
+
+#------------------------------------------------------------------------------
+# VERSION/PLATFORM COMPATIBILITY
+#------------------------------------------------------------------------------
+
+# - Previous PostgreSQL Versions -
+
+#array_nulls = on
+#backslash_quote = safe_encoding	# on, off, or safe_encoding
+#default_with_oids = off
+#default_with_rowids = off
+#escape_string_warning = on
+#lo_compat_privileges = off
+#quote_all_identifiers = off
+#sql_inheritance = on
+#standard_conforming_strings = on
+#synchronize_seqscans = on
+
+# - Other Platforms and Clients -
+
+#transform_null_equals = off
+
+
+#------------------------------------------------------------------------------
+# ERROR HANDLING
+#------------------------------------------------------------------------------
+
+exit_on_error = {{ pp_exit_on_error }}			# terminate session on any error?
+restart_after_crash = {{ pp_restart_after_crash }}		# reinitialize after backend crash?
+
+
+#------------------------------------------------------------------------------
+# CONFIG FILE INCLUDES
+#------------------------------------------------------------------------------
+
+# These options allow settings to be loaded from files other than the
+# default postgresql.conf.
+
+#include_dir = 'conf.d'			# include files ending in '.conf' from
+					# directory 'conf.d'
+#include_if_exists = 'exists.conf'	# include file only if it exists
+#include = 'special.conf'		# include file
+
+
+#------------------------------------------------------------------------------
+# CUSTOMIZED OPTIONS
+#------------------------------------------------------------------------------
+
+#dbms_pipe.total_message_buffer = 30kB  # default: 30KB, max: 256MB, min: 30KB
+#dbms_alert.max_alerts = 100		# default 100, max: 500, min: 0
+
+
+#---------------------------------------------------------------------------
+# RUNTIME INSTRUMENTATION AND TRACING
+#---------------------------------------------------------------------------
+
+#timed_statistics = {{ pp_timed_statistics }}			# record wait timings, defaults to on
+
+# Add settings for extensions here
+{% for setting in pp_extension_settings %}
+{{ setting.name }} = {{ setting.value }}
+{% endfor %}
```

Comment:
Don't we need to have sections for the edb variables somewhere in this file?  Wrapped with `{{ if install_edb }}`?